spec/classes/chrony_spec.rb

```diff
@@ -125,6 +125,9 @@
               it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*cmdallow 1\.2\.3\.4$}) }
               it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*cmddeny 1\.2\.3$}) }
               it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*cmdallow all 1\.2$}) }
+              it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*leapsecmode slew$}) }
```

Comment:
You need to add these params to the code block above:
```
context 'with some params passed in' do
let(:params) do
{
            queryhosts: ['192.168/16'],
            port: '123',
            cmdport: '257',	
            config_keys_mode: '0123',
            config_keys_owner: 'steve',
            config_keys_group: 'mrt',
            config_keys_manage: true,
            chrony_password: 'sunny',
            bindcmdaddress: ['10.0.0.1'],
            leapsecmode: 'slew',
            maxslewrate: '1000',
            smoothtime: '400 0.001',
}
end
```

Reply:
yes, missed that. Thanks.